teaching-platfrom/main.py

```diff
@@ -0,0 +1,22 @@
+import yaml
```

Comment:
it's a good idea to split imports to groups and separate them with blank lines:
```
standard imports

public pypi imports

your private modules
```

Reply:
Done.